content/en/authors/yagut-shakizada/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+# Display name
+title: "Yagut Shakizada"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+- "yagut-shakizada"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: Research Student
+
+# Organizations/Affiliations
+organizations:
+- name: Nara Institute of Science and Technology
+  url: "http://www.naist.jp"
+
+# Short bio (displayed in user profile at end of posts)
+bio: Hi! It's Yagut from Azerbaijan. The research student of Software Design and Analysis Laboratory at Division of Information Science, NAIST. 
```

Comment:
If you would like to display this short bio on your profile page, add following lines at the bottom of this file.

```
## Profile
Hi! It's Yagut from Azerbaijan. The research student of Software Design and Analysis Laboratory at Division of Information Science, NAIST.
```